spec/unit/installer/project_cache/project_cache_analyzer_spec.rb

```diff
@@ -161,6 +161,52 @@ module ProjectCache
             result.pod_targets_to_generate.should.equal([@monkey_lib])
             result.aggregate_targets_to_generate.should.equal(nil)
           end
+
+          it 'returns all pod targets that share the same #pod_name' do
+            subspec_target_1 = fixture_pod_target('matryoshka/matryoshka.podspec', false,
+                                                  {}, [], Pod::Platform.new(:ios, '6.0'), [], 'Foo')
+            subspec_target_2 = fixture_pod_target('matryoshka/matryoshka.podspec', false,
+                                                  {}, [], Pod::Platform.new(:ios, '6.0'), [], 'Bar')
+            subspec_pods = [subspec_target_2, subspec_target_1]
+            subspec_pods.each do |target|
+              @sandbox.pod_target_project_path(target.pod_name).mkpath
+              target.support_files_dir.mkpath
+            end
+
+            cache_key_by_aggregate_target_labels = {
+              subspec_target_1.label => TargetCacheKey.from_pod_target(subspec_target_1),
+              subspec_target_2.label => TargetCacheKey.from_cache_hash('BUILD_SETTINGS_CHECKSUM' => 'Blah'),
+            }
+            cache = ProjectInstallationCache.new(cache_key_by_aggregate_target_labels, @build_configurations, @project_object_version)
+            analyzer = ProjectCacheAnalyzer.new(@sandbox, cache, @build_configurations, @project_object_version, subspec_pods, [])
+            result = analyzer.analyze
+            result.pod_targets_to_generate.should.equal(subspec_pods)
+            result.aggregate_targets_to_generate.should.equal(nil)
+          end
+
+          it 'returns sibling pod target when adding a new subspec' do
```

Comment:
Not sure what this second test does. It passes when the sibling computation is disabled...

Reply:
It's independent but somewhat related to sibling targets. When adding a subspec there will be two new pod targets that the analyzer thinks are new. Just another spec case I thought of

Reply:
Okay, I still don't get it but it might be the late hour (2AM).

The test might need a comment.